### PR TITLE
feature/ads-967 Figma Storybook plugin additions

### PR DESCRIPTION
**Pull request by cblva**

### CODE REVIEW

File: public/styles/components/nav-primary/_nav-primary-bar.scss
Comment (kaseyllabelle) [inline public/styles/components/nav-primary/_nav-primary-bar.scss:1]: let's pull this into a separate PR & bug ticket

File: public/styles/components/_select.scss
Comment (kaseyllabelle) [inline public/styles/components/_select.scss:1]: let's pull this into a separate PR & bug ticket

File: src/components/InputCheckbox/InputCheckbox.stories.tsx [hunk 1]
```diff
@@ -176,3 +182,14 @@ export const GroupWithHelpAndError = ({ ...args }) => {
   );
 };
 GroupWithHelpAndError.storyName = 'Group with help and error';
+
+export const GroupDisabled = ({ ...args }) => {
+  return (
+    <Fieldset legend="Legend">
```
Comment (kaseyllabelle) [inline src/components/InputCheckbox/InputCheckbox.stories.tsx:188]: let's test that putting the disabled attr on the individual inputs vs the fieldset element has the same outcome

File: src/components/RadioGroup/RadioGroup.stories.tsx [hunk 2]
```diff
@@ -192,3 +206,31 @@ export const ButtonGroupWithHelpAndError: Story = {
     </RadioGroup>
   )
 };
+
+export const ButtonGroupDisabled: Story = {
+  name: 'Button group disabled',
+  render: (args) => (
+    <RadioGroup buttonGroup {...args}>
```
Comment (kaseyllabelle) [inline src/components/RadioGroup/RadioGroup.stories.tsx:213]: same re: testing w/ disabled on fieldset vs inputs

File: src/components/InputCheckbox/InputCheckbox.stories.tsx (hunk 1 above)
Comment (cblva) [reply src/components/InputCheckbox/InputCheckbox.stories.tsx:188]: Tested! Works on individual checkboxes and fieldset.

File: src/components/RadioGroup/RadioGroup.stories.tsx (hunk 2 above)
Comment (cblva) [reply src/components/RadioGroup/RadioGroup.stories.tsx:213]: Tested! Works for individual radios, radio buttons and fieldset.

**Review by misdreavus79 — APPROVED** [review]
LGTM
